### DISCUSSION TITLE
Add sortable for multiselect

### DISCUSSION THREAD

**Discussion by GalinaBublik** [filamentphp/filament#8711] [Ideas]
How i can add drag and drop sorter for multiselect? 

Investigate code but can't add 'livewire-sortable' to selected ((
Please help.

![image](https://github.com/filamentphp/filament/assets/10061993/436cb458-6aae-422a-ab54-275707234dba)

**Comment by jechazelle** [comment]
Did you find a solution @GalinaBublik ?

**Reply by GalinaBublik** [reply]
No(
